### PR TITLE
Add Hindi Translations

### PR DESCRIPTION
**Pull request by PartMan7**
Couldn't find any translation team on Smogon, so thought I'd start work on this. Thanks to Infinite Misery and Crow Music for offering to help out with this. I'll get it run past an RO once it's done.

And to think Hindi was my least-favourite subject in school...

### CODE REVIEW

File: translations/hindi.json [hunk 1]
```diff
@@ -85,62 +85,62 @@
...
-		"Get help with this": "",
-		"The user \"${targetUser.name}\" is locked and cannot be PMed.": "",
-		"On this server, you must be of rank ${groupName} or higher to PM users.": "",
-		"This user is blocking private messages right now.": "",
-		"This ${Config.groups[targetUser.group].name} is too busy to answer private messages right now. Please contact a different staff member.": "",
-		"If you need help, try opening a <a href=\"view-help-request\" class=\"button\">help ticket</a>": "",
-		"You are blocking private messages right now.": "",
+		"You are ${lockType} and can only private message members of the global moderation team. ${lockExpiration}": "आप ${lockType} हैं और केवल staff के सदस्यों को ही निजी संदेश भेज सकते हैं।",
+		"Get help with this": "इसके लिए सहायता",
+		"The user \"${targetUser.name}\" is locked and cannot be PMed.": "यह user ${} locked है और इसे निजी संदेश नहीं भेजा जा सकता।",
```
Comment (FlyingPhantom) [inline translations/hindi.json:123]:
```suggestion
		"The user \"${targetUser.name}\" is locked and cannot be PMed.": "यह user \"${targetUser.name}\" locked है और इसे निजी संदेश नहीं भेजा जा सकता।",
```

File: translations/hindi.json [hunk 2]
```diff
@@ -85,62 +85,62 @@
...
+		"On this server, you must be of rank ${groupName} or higher to PM users.": "इस server पर users को निजी सन्देश भेज्ने के लिये आप्को rank ${groupName} या उस्से बडा होना चाहिए।",
+		"This user is blocking private messages right now.": "इस user ने फिलहाल निजी संदेशों को निस्र्द्ध कर रखा है।",
+		"This ${Config.groups[targetUser.group].name} is too busy to answer private messages right now. Please contact a different staff member.": "यह ${Config.groups[targetUser.group].name} फिलहाल व्यस्त है। कृपया किसि अन्य staff को संपर्क करें।",
+		"If you need help, try opening a <a href=\"view-help-request\" class=\"button\">help ticket</a>": "अगर आपको सहायता की आवश्यकता हो, एक <a href=\"view-help-request\" class=\"button\">उपादान याचिका</a> बना लिजिये।",
```
Comment (FlyingPhantom) [inline translations/hindi.json:127]: change उपादान याचिका to टिकट (easy to understand)

File: translations/hindi.json [hunk 3]
```diff
@@ -85,62 +85,62 @@
...
+		"Get help with this": "इसके लिए सहायता",
+		"The user \"${targetUser.name}\" is locked and cannot be PMed.": "यह user \"${targetUser.name}\" locked है और इसे निजी संदेश नहीं भेजा जा सकता।",
+		"On this server, you must be of rank ${groupName} or higher to PM users.": "इस server पर users को निजी सन्देश भेज्ने के लिये आप्को rank ${groupName} या उस्से बडा होना चाहिए।",
+		"This user is blocking private messages right now.": "इस user ने फिलहाल निजी संदेशों को निस्र्द्ध कर रखा है।",
+		"This ${Config.groups[targetUser.group].name} is too busy to answer private messages right now. Please contact a different staff member.": "यह ${Config.groups[targetUser.group].name} फिलहाल व्यस्त है। कृपया किसि अन्य staff को संपर्क करें।",
+		"If you need help, try opening a <a href=\"view-help-request\" class=\"button\">help ticket</a>": "अगर आपको सहायता की आवश्यकता हो, एक <a href=\"view-help-request\" class=\"button\">उपादान याचिका</a> बना लिजिये।",
+		"You are blocking private messages right now.": "आप्ने फिलहाल निजी संदेशों को निस्र्द्ध कर रखा है।",
```
Comment (FlyingPhantom) [inline translations/hindi.json:128]:
```suggestion
		"You are blocking private messages right now.": "आपने फिलहाल निजी संदेशों को निस्र्द्ध कर रखा है।",
```

**Comment by AnnikaCodes** [comment]
@PartMan7 are you still working on this?

**Comment by PartMan7** [comment]
Shelf life expired

**Pull request closed without merging**